modules/op-using-pipelines-as-code-with-a-github-app.adoc

```diff
@@ -39,7 +39,13 @@ To create and configure a GitHub App manually for {pac}, perform the following s
 
 * **GitHub Application Name**: `{pipelines-shortname}`
 * **Homepage URL**: OpenShift Console URL 
-* **Webhook URL**: The {pac} route or ingress URL. You can find it by running the command `echo https://$(oc get route -n openshift-pipelines pipelines-as-code-controller -o jsonpath='{.spec.host}')`.
+* **Webhook URL**: The {pac} route or ingress URL. You can find it by running the following command:
++
+[source,terminal]
+----
+$ echo https://$(oc get route -n openshift-pipelines pipelines-as-code-controller -o jsonpath='{.spec.host}')
+----
+
 * **Webhook secret**: An arbitrary secret. You can generate a secret by executing the command `openssl rand -hex 20`.
```

Comment:
How about this? Do we need to put the command `openssl rand -hex 20` like the previous one?

Reply:
@Srivaralakshmi You are right. Thank you for noticing!